packages/renderer/src/lib/container/ContainerColumnAge.svelte

```diff
@@ -0,0 +1,12 @@
+<script lang="ts">
+import StateChange from '../ui/StateChange.svelte';
+import type { ContainerGroupInfoUI, ContainerInfoUI } from './ContainerInfoUI';
+
+export let object: ContainerInfoUI | ContainerGroupInfoUI;
+</script>
+
+{#if 'state' in object}
```

Comment:
```suggestion
{#if 'state' in object && typeof object.state === 'string'} 
```

Reply:
I am wondering if typescript should not complain here as state would be unknown type before checking its type

Reply:
Only ContainerInfoUI has state and it's a string there though?

Reply:
what if I am sending you a { state: true }

Reply:
As per other comment, I've added isContainerGroupInfoUI/isContainerInfoUI helper utils which do this.